### PR TITLE
Fix #228

### LINKED ISSUE #228: Resolve TODO: a= lines that must be processed
http://rtcweb-wg.github.io/jsep/#rfc.section.5.6.1
[TODO: identity, rtcp-rsize, rtcp-mux, and any other attribs valid at session level.]

### PR DESCRIPTION
**Pull request by fluffy**
Fixes #228. Not ready yet.

### CODE REVIEW

File: draft-ietf-rtcweb-jsep.xml [hunk 1]
```diff
@@ -3349,10 +3349,18 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
                         <t>Any "a=extmap" lines are parsed as specified
                         in <xref target="RFC5285" />, Section 5, and
                         their values are stored.</t>
-
-                        <t>TODO: identity, rtcp-rsize, rtcp-mux, and any
-                        other attributes valid at session level.</t>
```
Comment (fluffy) [inline draft-ietf-rtcweb-jsep.xml]: Note , Did we mean fingerprint when we said "identity" in above TODO

File: draft-ietf-rtcweb-jsep.xml [hunk 2]
```diff
@@ -3349,10 +3349,18 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
...
+			
+			<t>Any "a=rtcp-rsize" lines are parsed as specified in
+			<xref target="RFC5506" />, Section 5, and their
+			values are stored.</t>
+
+			<t>Any "a=rtcp-mux" lines are parsed as specified in
+			<xref target="RFC5761" />, Section 5, and their
+			values are stored.</t>
+
+		<!--	<t> TODO -  are there any other attributes
```
Comment (ekr) [inline draft-ietf-rtcweb-jsep.xml]: What about mux-only

File: draft-ietf-rtcweb-jsep.xml [hunk 3]
```diff
@@ -3358,6 +3358,11 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
 			<xref target="RFC5761" />, Section 5, and their
 			values are stored.</t>
 
+			<t>Any "a=rtcp-mux-only" lines are parsed as
+			specified in <xref
+			target="I-D.ietf-mmusic-mux-exclusive" />,
+			Section 3, and store this value.</t>
```
Comment (ekr) [inline draft-ietf-rtcweb-jsep.xml]: and their values are stored.

**Comment by ekr** [comment]
LGTM with editorial comment as above.

File: draft-ietf-rtcweb-jsep.xml [hunk 4]
```diff
@@ -3349,10 +3349,23 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
...
                         in <xref target="RFC5285" />, Section 5, and
                         their values are stored.</t>
-
-                        <t>TODO: identity, rtcp-rsize, rtcp-mux, and any
-                        other attributes valid at session level.</t>
-
+			
+			<t>Any "a=rtcp-rsize" lines are parsed as specified in
+			<xref target="RFC5506" />, Section 5, and their
+			values are stored.</t>
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: These should be analogous to the text describing the handling of rsize, mux at the media level, which uses the following construction:
    If present, a single "a=rtcp-mux" attribute MUST be parsed as
    specified in [RFC5761], Section 5.1.1, and its presence or absence
    flagged and stored.

Suggest we also use the same ordering, i.e. mux, then rsize.

File: draft-ietf-rtcweb-jsep.xml [hunk 5]
```diff
@@ -3349,10 +3349,23 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
...
+
+			<t>Any "a=rtcp-mux" lines are parsed as specified in
+			<xref target="RFC5761" />, Section 5, and their
+			values are stored.</t>
+
+			<t>Any "a=rtcp-mux-only" lines are parsed as
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: Can you add this at media level too

**Comment by juberti** [comment]
Please apply same presence/absence treatment for all

**Pull request merged**